bin/deploy-release.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -ex
+set -eo pipefail
+
+# Strip the 'v' from the Tag Name
+export TAG=${TAG_NAME:1}
```

Comment:
There's a better way to do this with something like `${TAG_NAME#$v}`

Reply:
You're correct - did some googling and went with:
```
export TAG=${TAG_NAME//"v"}
```